Tools/CISupport/ews-build/steps.py

```diff
@@ -4375,7 +4375,7 @@ def run(self):
 
         rc = yield super(RunAPITests, self).run()
 
-        if rc == SUCCESS:
+        if rc in [SUCCESS, WARNINGS] and self.name != RunAPITestsWithoutChange:
```

Comment:
RunAPITestsWithoutChange => RunAPITestsWithoutChange.name

Ideally we should have 'Passed API tests' message even for RunAPITestsWithoutChange. maybe the if check for self.name can be only for buildFinished statement.